src/librustc_span/symbol.rs

```diff
@@ -856,8 +856,6 @@ symbols! {
         rustc_unsafe_specialization_marker,
         rustc_variance,
         rust_eh_personality,
```

Comment:
should this have been removed?

Reply:
Its one use point was removed, so, seems reasonable.